coordinator/gscoordinator/kubernetes_launcher.py

```diff
@@ -192,10 +198,50 @@ def __init__(
         self._mars_worker_cpu = mars_worker_cpu
         self._mars_worker_mem = mars_worker_mem
 
+        # check the validity of mode
+        self._mode = mode
+        if self._mode not in ["eager", "lazy"]:
+            logger.error(
+                "Invalid mode: %s, mode must be one of eager or lazy", self._mode
+            )
```

Comment:
If mode is not recognized, set the mode to the default ('eager').
also log it: `logger.error('Invalid mode %s, choose from 'eager' or 'lazy'. Proceeding with default mode: 'eager')`